scripts/cli/src/helpers/rewards_helper.ts

```diff
@@ -0,0 +1,36 @@
+import type { KeyringPair } from "@polkadot/keyring/types";
+import type { AccountId } from "@polkadot/types/interfaces/runtime";
+import { sendTx, ApiSingleton } from "../util/init";
+import { ItnRewardStatus } from "../types";
+import { u8aToHex, numberToU8a } from "@polkadot/util";
+
+/**
+ * @description Claim an ITN reward with a valid signature.
+ */
+export async function claimItnReward(
+	signer: KeyringPair,
+	itnAddress: string | Uint8Array | AccountId
```

Comment:
Instead of the `itnAddress` pass in the signer object, because it will be needed to sign the claim message.